src/Stratis.FederatedPeg.Tests/CrossChainTransferStoreTests.cs

```diff
@@ -92,17 +97,17 @@ public void SynchronizeSynchronizesWithChainAndSurvivesRestart()
         {
             ConcurrentChain chain = BuildChain(5);
 
-            string dataDir = GetTestDirectoryPath(this);
+            string dataDir = CreateTestDir(this);
+            var dataFolder = new DataFolder(dataDir);
             var nodeSettings = new NodeSettings(this.network, NBitcoin.Protocol.ProtocolVersion.ALT_PROTOCOL_VERSION,
```

Comment:
As discussed, for now OK, but if we can avoid using NodeSettings and instead use mocked IFederationGatewaySettings then we shouldn't have tests randomly failing

Reply:
Done